### PR TITLE
feat(sso): SAML setup instructions with Ping Identity

### PR DESCRIPTION
**Pull request by leedongwei**

### CODE REVIEW

File: src/docs/product/accounts/sso/ping-sso.mdx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+1. Log in to [Ping Identity](https://www.pingidentity.com/en/account/sign-on.html).
+
+1. Go to "Adminstrators" environment for your organization.
+
+1. If you don't require a permission group for Sentry, skip to Step 5.
+
+1. In the sidebar, click on "Identities" then "Groups", and navigate to it. From there, create a new group, add yourself and other members to it. For more details about group creation, see the [Ping docs](https://docs.pingidentity.com/bundle/pingone/page/vdz1610472224361.html).
+
+1. In the sidebar, click on "Connections" then "Applications". Click on "+ Application" and for the application type, select "Web App" and "SAML".
```
Comment (imatwawana) [inline src/docs/product/accounts/sso/ping-sso.mdx:17]:
```suggestion
1. In the sidebar, click on "Connections", then "Applications". Click on "+ Application" and for the application type, select "Web App" and "SAML".
```

File: src/docs/product/accounts/sso/ping-sso.mdx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+   ![Create web application for Ping](ping-new-app.png)
+
+1. For Section (1), labeled "Create App Profile", you can get the Sentry logo from [here](https://sentry.io/branding/).
+
+1. For Section (2), labeled "Configure SAML", select "Manually Enter". Enter the following data in each line and save your changes.
```
Comment (imatwawana) [inline src/docs/product/accounts/sso/ping-sso.mdx:23]: If the word "Section" isn't capitalized in the Ping UI, then don't capitalize it here.

File: src/docs/product/accounts/sso/ping-sso.mdx [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+   - Assertion Consumer Service (ACS) URL: `https://sentry.io/saml/acs/YOUR_ORG_SLUG/`
+
+   - Entity ID: `https://sentry.io/saml/metadata/YOUR_ORG_SLUG/`
+
+   - Single-Logout (SLO) Endpoint: `https://sentry.io/saml/sls/YOUR_ORG_SLUG/`
+
+   - Target Application URL: `https://sentry.io/organizations/YOUR_ORG_SLUG/`
+
+1. For Section (3), labeled "Attribute Mappings". For "Application Attribute" ← "Outgoing Value", use:
```
Comment (imatwawana) [inline src/docs/product/accounts/sso/ping-sso.mdx:33]: should that be a left arrow?

```suggestion
1. In Section (3), labeled "Attribute Mappings", for "Application Attribute" ← "Outgoing Value", use:
```

File: src/docs/product/accounts/sso/ping-sso.mdx [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+   - `saml_subject` ← `Email address`
+
+   - `email` ← `Email address`
+
+   - `first_name` ← `Given Name`
+
+   - `last_name` ← `Family Name`
+
+     You can use custom key values, but you will need to amend the steps below. If you would like to use custom attribute mapping, see
```
Comment (imatwawana) [inline src/docs/product/accounts/sso/ping-sso.mdx:43]: Just highlighting this as needing completion

```suggestion
     You can use custom key values, but you will need to amend the steps below. If you would like to use custom attribute mapping, see
```

File: src/docs/product/accounts/sso/ping-sso.mdx [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+1. Once the application is created, click on the Sentry application and navigate to "Configuration" and copy "IDP Metadata URL"
+
+   ![Ping configuration for Sentry](ping-configure-sentry.png)
+
+1) Go back to Sentry. Navigate to your **Org Settings > Auth** (or go to `https://sentry.io/settings/YOUR_ORG_SLUG/auth/`) and click on "Configure" for SAML2.
```
Comment (imatwawana) [inline src/docs/product/accounts/sso/ping-sso.mdx:49]:
```suggestion
1. Go back to [sentry](https://sentry.io). Navigate to **Settings > Auth** (or go to `https://sentry.io/settings/YOUR_ORG_SLUG/auth/`) and click on "Configure" for SAML2.
```

File: src/docs/product/accounts/sso/ping-sso.mdx [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+1) Paste the IDP Metadata URL from above and click "Get Metadata".
```
Comment (imatwawana) [inline src/docs/product/accounts/sso/ping-sso.mdx:51]:
```suggestion
1. Paste the IDP Metadata URL from above and click "Get Metadata".
```

File: src/docs/product/accounts/sso/ping-sso.mdx [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+
+1) In the next page, enter the following keys in their respective fields to map the Ping ID attributes to Sentry, and then save them.
```
Comment (imatwawana) [inline src/docs/product/accounts/sso/ping-sso.mdx:53]:
```suggestion
1. On the next page, enter the following keys in their respective fields to map the Ping ID attributes to Sentry, and then save them:
```

File: src/docs/product/accounts/sso/ping-sso.mdx [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+
+   - User Email: `email`
+
+   - First Name: `first_name`
+
+   - Last Name: `last_name`
+
+     For more details about mappings for custom configs, see [this guide on Ping](https://support.pingidentity.com/s/article/PingOne-Using-Advanced-Attribute-Mapping).
+
+1) Sentry will attempt to authenticate and link your account with Ping. After successful authentication, you'll be redirected to Sentry's SSO configuration page, where you can take the following actions:
```
Comment (imatwawana) [inline src/docs/product/accounts/sso/ping-sso.mdx:65]:
```suggestion
1. Sentry will attempt to authenticate and link your account with Ping. After successful authentication, you'll be redirected to Sentry's SSO configuration page, where you can take the following actions:
```

File: src/docs/product/accounts/sso/ping-sso.mdx [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+
+   - You can share the "Login URL" value, which will be used for SP-initiated SSO, with the users in your organization.
+
+   - Scroll down to the bottom and ensure that "Require SSO" is checked if you want to enforce logging in with Ping.
+
+   - Set a "Default Role" for new SSO users. Selecting "Member" should cover most use-cases.
```
Comment (imatwawana) [inline src/docs/product/accounts/sso/ping-sso.mdx:71]:
```suggestion
   - Set a "Default Role" for new SSO users. Selecting "Member" should cover most use cases.
```

**Review by imatwawana — APPROVED** [review]
Some minor wording and style fixes.

File: src/docs/product/accounts/sso/ping-sso.mdx (hunk 3 above)
Comment (leedongwei) [reply src/docs/product/accounts/sso/ping-sso.mdx]: Yea, that's the direction of the arrow in Ping's UI.

**Review by JoshFerge — APPROVED** [review]

**Pull request merged**